### ISSUE TITLE
Fatal exception: Depleted resources

### ISSUE THREAD

**Issue by pepopowitz** [camunda/camunda-platform-docs#1129]
# TL;DR 

I am out of Cheetos.

![ezgif com-gif-maker](https://user-images.githubusercontent.com/1627089/183514615-81795d82-6ecb-4988-9a27-b0c15f37d2d5.gif)

## Background

When I started at Camunda [I had 40 bags of Cheetos](https://twitter.com/pepopowitz/status/1509632600108941316). As of this afternoon, I have 0 bags of Cheetos. 

Yes I have only eaten on average 10 bags per month, but every one of them was delicious and now I miss them.

## Severity

Very very high

## How does this issue affect developers

I am sad, lonely, confused, and hungry.

* Note that developers other than me (@pepopowitz) are likely not affected by this issue.

## How does this issue affect users

It doesn't, at least not directly. Long term maybe we could argue that me not having snacks leads to poor decisions leads to a poor docs experience. 

## How is this issue resolved

![moar snaks](https://media.giphy.com/media/Utts4yxFGpQbkvwamq/giphy.gif)

**Comment by akeller** [comment]
_Minding my own business working down all my notifications and triaging things for Q3, Q4, and beyond._
_Gets distracted trying to book travel for November._
_Checks Trello before getting back to the things I actually need to do today..._
"WHAT IS THIS....?!"

And then I saw the watch and was like "he's over there making his own gifs" 😆

**Comment by pepopowitz** [comment]
Wow my watch is part of my brand at this point????!!!

Also if you're wondering, YES, I have been _aching_ to open this issue for weeks, since I first noticed I was getting low on supplies.

**Comment by akeller** [comment]
> "I have 11 more Skittles left of the 3lb+ bag you sent before I onboarded."
- Me to Mary on March 16, 2021

I onboarded Nov 1, 2020.

**Comment by pepopowitz** [comment]
Happy to see this is a tradition

**Comment by akeller** [comment]
Is there a business need to send you more Cheetos? 🤔

**Comment by pepopowitz** [comment]
> Long term maybe we could argue that me not having snacks leads to poor decisions leads to a poor docs experience.

Won't someone think about the users?!?!?!
